### DISCUSSION TITLE
GrapesJS Doesn't Work Even In Official Documents

### DISCUSSION THREAD

**Discussion by bforbiggy** [GrapesJS/grapesjs#5944] [Q&A]
![image](https://github.com/GrapesJS/grapesjs/assets/22730639/29dd89f7-c301-4996-b312-6dc4957d368e)

The title is pretty self-explanatory, with https://grapesjs.com/docs/getting-started.html#start-from-the-canvas it's supposed to already have some base functionality. Just from the first section of starting from the canvas, it says "you're already able to move, copy and delete components". 

From the screenshot and installing it through npm, this is not the case. It just doesn't load please help!!!!

**Comment by bforbiggy** [comment]
Quick update, it works on local development. I was simply initializing too early before it could find the element to initialize grapesjs, needed to use Sveltekit's onMount function.

However, the official documents are still broken and I don't know why

**Reply by Isc-mntl-snchz** [reply]
If you share a bit of code or screenshot of what is wrong, we could respond. Once loaded, the canvas is even available to paste text or simply write on. In fact, you can add components from your Blocks, or remove them.

**Reply by bforbiggy** [reply]
> If you share a bit of code or screenshot of what is wrong, we could respond. Once loaded, the canvas is even available to paste text or simply write on. In fact, you can add components from your Blocks, or remove them.

Maybe I said it in a confusing way but my original comment meant to say that grapesjs works on my own website but not on official docs. My issue now is just with the official website which I linked and screenshotted, because while the library works when I use it, the quick start guide on grapesjs.com doesn't work.

**Comment by artf** [comment]
Thanks for the report @bforbiggy Fixed the examples in docs

**Comment by bforbiggy** [comment]
It bizarrely doesn't work on Vivaldi but it works on chrome/firefox now and that's more than good enough.
Thanks so much!
